adafruit_motor/stepper.py

```diff
@@ -113,8 +113,13 @@ def __init__(
             # set a safe pwm freq for each output
             self._coil = (ain2, bin1, ain1, bin2)
             for i in range(4):
-                if self._coil[i].frequency < 1500:
+                if self._coil[i].frequency < 1500 and self._coil[i].variable_frequency:
                     self._coil[i].frequency = 2000
+                else:
```

Comment:
Won't this error fire if the frequency is above 1500 but not variable? Or do I misread it?

Reply:
You are absolutely correct, the logic should be inverted to use a guard clause instead for `if under and not variable: raise error`.